### PR TITLE
add origin story, minor tweaks from #27. closes #27

### LINKED ISSUE #27: Add explanation of Link-State conference name
I know it has something to do with the link-state routing protocol and I remember it was really cute, but the connection between the two escapes me. Nobody else seems to know it, either, so someone who remembers the connection should add it to the conference page so that it won't be forgotten over the years :) http://acm.case.edu/acm/conference/about

### PR DESCRIPTION
**Pull request by ajm188**

### CODE REVIEW

**Comment by thomasrussellmurphy** [comment]
Incomplete hard wrap in first edited paragraph?

**Comment by ajm188** [comment]
I don't think so. Point out the line # ?

File: app/views/acm/conference/about.haml [hunk 1]
```diff
@@ -3,8 +3,53 @@
 %p
   Link-State is a student run conference put on every year by Case Western's ACM
   student group. The first Link-State conference was run October 27th 2012 and it
-  has been growing ever since. If you want to see what was happening at our previous
-  conferences, check out the archive below.
+  has been growing ever since. If you're curious as to where the name came from,
+  read its origin story below. If you want to see what was happening at our previous
+  conferences, check out the
```
Comment (thomasrussellmurphy) [inline app/views/acm/conference/about.haml]: This and following two lines.

File: app/views/acm/conference/about.haml [hunk 2]
```diff
@@ -3,8 +3,53 @@
...
+  %a{ href: '#archive' } archive
+  further down the page.
```
Comment (thomasrussellmurphy) [inline app/views/acm/conference/about.haml]: Above three lines. Excuse the deletion.

Comment (ajm188) [inline app/views/acm/conference/about.haml]: I thought that's how you put links inside of p tags in haml. I'll try doing it inline.

**Comment by thomasrussellmurphy** [comment]
A `#{link_to("display text", "#destination_anchor")}` may work to smooth the line.

**Comment by ajm188** [comment]
Oh yeah, I forget I just get to sprinkle ruby wherever I please. *facedesk*

**Comment by thomasrussellmurphy** [comment]
Righty-o. Looks like that's everything.

**Pull request merged**